Doc/howto/instrumentation.rst

```diff
@@ -57,18 +64,15 @@ On macOS, you can list available DTrace probes by running a Python
 process in the background and listing all probes made available by the
 Python provider::
 
-   $ python3.6 -q &
-   $ sudo dtrace -l -P python$!  # or: dtrace -l -m python3.6
+   $ python3.12 -q &
```

Comment:
I don't like the mention to `python3.12` here. Can't stick to `python` only? The rationale is that Python 3.12 is security-only. I would prefer that we use the correct version (for main it would be Python 3.15, etc)